# Exception_r

## goto Project.yml 
### line no 8 use_exceptions: TRUE

## when apply Try Catch cant have TEST_ASSERT_....
